docs/user-docs/latest/content-manager/translating-content.md

```diff
@@ -0,0 +1,25 @@
+# Translating content
+
+With the Internationalization plugin installed (see [Installing plugins via the Marketplace](../plugins/installing-plugins-via-marketplace.md)), it is possible to manage content in more than one language, called "locale". To manage content in a specific locale, the latter must be added beforehand through the Internationalization settings (see [Configuring Internationalization locales](../settings/managing-global-settings.md#configuring-internationalization-locales)).
+
+In the Content Manager, when the Internationalization plugin is installed, some options are added to the edit view:
+
+- an Internationalization box, displayed in the right side of the interface, from where it is possible to switch locales
+- icons displayed next to every field to indicate whether the field can be translated or not:
+  - <img width="16" src="../assets/content-manager/icon_localizable.png"> indicates that the field can be translated,
+  - <img width="18" src="../assets/content-manager/icon_non-localizable.png"> indicates that the field cannot be translated: its content is managed from the default locale only and will be the same for every locale.
```

Comment:
> indicates that the field cannot be translated: its content is managed from the default locale only and will be the same for every locale.

I am not sure about this statement as non localised field can be managed in all locales, however the value is common for all languages, modifying the value in one locale will update all the others.